e2e/run_tests.js

```diff
@@ -161,7 +161,22 @@ function printMessage(testFiles, index) {
 
     // Log which files were being tested
     console.log(chalk.magenta.bold(`${invert ? 'All Except --> ' : ''}${testStage}${stage && withGroup ? '| ' : ''}${testGroup}`));
-    console.log(chalk.magenta(`(Testing ${index + 1} of ${testFiles.length})  - `, testFile));
+    console.log(chalk.magenta(`(Testing ${overallIndex + 1} of ${testFiles.length})  - `, testFile));
+    if (process.env.CI_BASE_URL) {
+        console.log(chalk.magenta(`Testing ${currentIndex}/${lastIndex} in "${process.env.CI_BASE_URL}" server`));
+    }
```

Comment:
This will generate log, like
![Screen Shot 2020-12-01 at 8 55 05 PM](https://user-images.githubusercontent.com/5334504/100743720-aad65f00-3417-11eb-871a-b76aafd6f304.png)